sub-packages/bionemo-geneformer/tests/bionemo/geneformer/test_model.py

```diff
@@ -684,7 +684,8 @@ def _get_loss_from_model(model_config: GeneformerConfig, seed: int) -> float:
                 attention_mask=batch["attention_mask"].cuda(),
             )
             loss_mask = batch["loss_mask"].cuda()
-            logits = result["token_logits"]
+            # token_logits is s,b and for simplicity here let's transpose to b,s. In general this reduces performance.
+            logits = result["token_logits"].transpose(0, 1).contiguous()
             target = batch["labels"].cuda()
 
             loss += F.cross_entropy(logits[loss_mask].float(), target[loss_mask], reduction="sum")
```

Comment:
How about reusing `unreduced_token_loss_fn` if we trust that function?

Reply:
I would then have to sum reduce myself? I'd rather leave it like this for now.